network/onchain-discovery/src/test.rs

```diff
@@ -160,25 +161,29 @@ fn gen_configs(count: usize) -> Vec<NodeConfig> {
 
 fn setup_storage_service_and_executor(
     config: &NodeConfig,
-) -> (Runtime, Arc<dyn StorageRead>, Executor<LibraVM>, Waypoint) {
-    let (arc_db, db_reader_writer) = init_libra_db(config);
+) -> (
+    ThreadJoinHandle<()>,
+    Arc<dyn DbReader>,
+    Executor<LibraVM>,
+    Waypoint,
+) {
+    let (db_r, db_rw) = init_libra_db(config);
     let genesis_tx = config.execution.genesis.as_ref().unwrap();
-    let waypoint = bootstrap_db_if_empty::<LibraVM>(&db_reader_writer, genesis_tx)
+    let waypoint = bootstrap_db_if_empty::<LibraVM>(&db_rw, genesis_tx)
         .expect("Db-bootstrapper should not fail.")
         .unwrap();
-    let storage_runtime = start_storage_service_with_db(config, arc_db);
 
-    let storage_read_client = Arc::new(StorageReadServiceClient::new(&config.storage.address));
-    let executor = Executor::new(SyncStorageClient::new(&config.storage.address).into());
+    let storage_thread = start_simple_storage_service_with_db(config, Arc::clone(&db_r));
+    let executor = Executor::new(SimpleStorageClient::new(&config.storage.simple_address).into());
```

Comment:
we shouldn't be using simple_storage_service afaiu, we should just be getting a direct handle to the db_r and passing that in: https://github.com/libra/libra/blob/master/secure/key-manager/src/tests.rs#L299

Reply:
yeah no need to start the storage service in tests, use db directly.